run_sample_httpserver.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
```

Comment:
Rename the script `httpserver` -> `standalone`.

Reply:
Fixed.